server/modules/resourceFactories/AsgResource.js

```diff
@@ -104,6 +104,10 @@ function AsgResource(client, accountName) {
       request.LaunchConfigurationName = parameters.launchConfigurationName;
     }
 
+    if (!_.isNil(parameters.subnets)) {
```

Comment:
I know this returns boolean, but can we still use strict equals as a good coding habit? `_.isNil === false`